doc/source/whatsnew/v1.1.0.rst

```diff
@@ -215,6 +215,50 @@ MultiIndex
         # Common elements are now guaranteed to be ordered by the left side
         left.intersection(right, sort=False)
 
+- Bug in :meth:`MultiIndex.get_indexer` incorrectly handling use of pad and backfill options.  This is a regression introduced in pandas 0.23.0 (:issue:`29896`)
```

Comment:
don't put this in bug fixes, this needs a separate section in api breaking changes), with a sub-header, see other examples there.

Reply:
done